style.css

```diff
@@ -430,3 +429,161 @@ div.border-bootom {
   color: #fff;
   background: #2230d2;
 }
+
+.navbar .links {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    margin: 10px 20px;
+    padding: 0 60px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navbar .links {
+    display: flex;
+  }
+
+  ul.links {
+    margin-top: 0;
+  }
+
+  .main-section {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    padding: 50px 130px;
+  }
+
+  .main-section h1 {
+    grid-column-start: 1;
+    grid-column-end: 3;
+  }
+
+  .main-section p {
+    grid-column-start: 1;
+    grid-column-end: 4;
+  }
+
+  .main-section div {
+    grid-column-start: 1;
+    grid-column-end: 2;
+  }
+
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .grid-2 {
+    display: grid;
+    order: 2;
+  }
+
+  .project .project-img {
+    width: 100%;
+    max-height: 100%;
+    height: auto;
+  }
+
+  .prject-content {
+    margin: 20px;
+    padding-bottom: 41px;
+  }
+
+  div.border-bootom {
+    border-bottom: 1px solid #dfe1e6;
+    padding: 20px;
+  }
+
+  .languages li {
+    margin: 20px 0;
+    justify-content: space-between;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    display: block;
+    color: #000;
+  }
+
+  .languages div:first-child {
+    padding-right: 20px;
+  }
+
+  .mycompetence li {
+    margin: 10px;
+    align-items: center;
+    max-width: 100px;
+    height: 100px;
+    background: #f7f7f9;
+    border-radius: 8px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    color: #253858;
+  }
+
+  .project .toni {
+    justify-content: flex-start;
+  }
+
+  .languages ul.mycompetence {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .languages:first-child {
+    padding: 5px 20px;
+  }
+
+  .about {
+    margin: 30px 5px;
+    background-color: #fff;
+    padding: 20px 16px;
+  }
+
+  .about .content {
+    padding: 20px 0;
+  }
+
+  .about p {
+    color: #172b4d;
+    padding-right: 20px;
+  }
+
+  .content .links {
+    padding: 20px 5px;
+  }
+
+  .contact-form {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  .contac-card {
+    grid-column: 2/4;
+  }
+
+  .contact-me-form .form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
- Good job on the work done so far, but there are some differences between your project and the chosen design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project(Please pay attention to the marked section in the Figma template attached below):
- The background images in the section is pixelated, please try to fix this 🙏 
- The section contains the whole width of the screen, there are no white borders on the left and right as can be seen in your design, please fix this in your design 👍 👍 

<img width="946" alt="eththet" src="https://user-images.githubusercontent.com/73607512/207820687-6067a0f2-7133-4d93-9c01-22b6d37475b9.PNG">

Figma design template:
<img width="566" alt="hwrwrg" src="https://user-images.githubusercontent.com/73607512/207821537-12d09d0e-bcc5-47e4-8e85-d3a817baa0fc.PNG">

Reply:
Fixed.